packages/manager/modules/pci/src/projects/project/workflow/add/resources/resources.html

```diff
@@ -71,3 +71,11 @@
     >
     </oui-select-picker>
 </div>
+<oui-message
+    data-ng-if="$ctrl.selectedResource.name.indexOf('i1') === 0"
```

Comment:
This could also be replaced by a `isIOPS()` check

Reply:
Here `selectedResource` is an instance. I think you should avoid test the instance `name`, because the `name` is defined by the customer.

Reply:
Ok thanks for this info

Reply:
I will try to use another way to fix this

Reply:
Please take a look at my changes